.github/workflows/rust.yml

```diff
@@ -60,7 +60,7 @@ jobs:
       - run: just wait-for-prepull
       - name: Run heavy tests
         run: |
-          cargo test --release \
+          TEST_LOG=true RUST_LOG=info cargo test --release \
```

Comment:
does this need to be taken out before merging into main?

Reply:
yeah i'd take it out, just using it for debugging rn